de.fu_berlin.inf.dpp.intellij/src/de/fu_berlin/inf/dpp/intellij/session/SessionUtils.java

```diff
@@ -0,0 +1,93 @@
+package de.fu_berlin.inf.dpp.intellij.session;
+
+import de.fu_berlin.inf.dpp.SarosPluginContext;
+import de.fu_berlin.inf.dpp.activities.SPath;
+import de.fu_berlin.inf.dpp.filesystem.IResource;
+import de.fu_berlin.inf.dpp.session.ISarosSession;
+import de.fu_berlin.inf.dpp.session.ISarosSessionManager;
+import de.fu_berlin.inf.dpp.session.NullSessionLifecycleListener;
+import de.fu_berlin.inf.dpp.session.SessionEndReason;
+
+import org.jetbrains.annotations.NotNull;
+import org.picocontainer.annotations.Inject;
+
+/**
+ * A utility class to provide easy access to functionality needing a session
+ * object.
+ * <p>
+ * <b>NOTE:</b> It should be carefully considered when to use this class as it
+ * generally is an indicator that the calling class should belong to the session
+ * context (which would make the usage of this class unnecessary).
+ * </p>
+ *
+ * @deprecated Consider whether your class actually belongs inside the session
+ * context and use the session container instead in such cases.
+ */
+//TODO consider to remove this class after the session & plugin context have been correctly separated
+@Deprecated
+public class SessionUtils {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Do not forget to remove this deprecated code someday. ([squid:S1133](https://sonarcloud.io/organizations/saros/rules#rule_key=squid%3AS1133))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=saros_complete&pullRequest=293&issues=AWcxWGhR0POp3Yiz0EfM&open=AWcxWGhR0POp3Yiz0EfM)

[ISSUE_KEY]: AWcxWGhR0POp3Yiz0EfM